### DISCUSSION TITLE
Defining scope of collaboration with GSMA OPAG

### DISCUSSION THREAD

**Discussion by sergiofranciscoortiz** [camaraproject/EdgeCloud#182] [Q&A]
This issue is to open a discussion on  the scope of collaboration from Operator Platform API Group (OPAG) and which necessities from CAMARA EdgeCloud group may be covered by OPAG.

**Comment by Kevsy** [comment]
HI @sergiofranciscoortiz , do you have any more details please? e.g. What sort of necessities do you think may fall under OPAG specifically for EdgeCloud..?
